sample/prometheus-cm1-configmap.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+data:
+  targets.yml: |
```

Comment:
Can we make the file name better, like `Prometheus-targets-cm.yaml`, same with other files

Reply:
The filename is generated by kompose, it will be improved later.